File: jdroid-android-firebase-remoteconfig/src/debug/java/com/jdroid/android/firebase/remoteconfig/FirebaseRemoteConfigActivity.java
package com.jdroid.android.firebase.remoteconfig;

import androidx.fragment.app.Fragment;

import com.jdroid.android.activity.FragmentContainerActivity;

public class FirebaseRemoteConfigActivity extends FragmentContainerActivity {

	@Override
	protected Class<? extends Fragment> getFragmentClass() {
		return FirebaseRemoteConfigFragment.class;
	}
}

File: jdroid-android-firebase-remoteconfig/src/debug/java/com/jdroid/android/firebase/remoteconfig/FirebaseRemoteConfigActivity.kt
package com.jdroid.android.firebase.remoteconfig

import androidx.fragment.app.Fragment

import com.jdroid.android.activity.FragmentContainerActivity

class FirebaseRemoteConfigActivity : FragmentContainerActivity() {

    override fun getFragmentClass(): Class<out Fragment> {
        return FirebaseRemoteConfigFragment::class.java
    }
}
